src/snowflake/cli/api/project/definition_conversion.py

```diff
@@ -130,7 +131,10 @@ def convert_project_definition_to_v2(
         # since the file won't be re-read as it would be for a permanent conversion
         definition_v2 = render_definition_template(data, {}).project_definition
     else:
-        definition_v2 = ProjectDefinitionV2(**data)
+        # Exact values of context are irrelevant.
```

Comment:
I don't get this either - didn't we make this irrelevant because we don't check the value later? We could actually check for the "conversion_in_memory" key later, which makes the value of the context relevant.

Reply:
Changed